pkg/signatures/bbs/pok_signature_proof_test.go

```diff
@@ -61,6 +61,9 @@ func TestPokSignatureProofSomeMessagesRevealed(t *testing.T) {
 	pok, err := NewPokSignature(sig, generators, proofMsgs, crand.Reader)
 	require.NoError(t, err)
 	require.NotNil(t, pok)
+	gens, blinds := pok.GetBlindingValues()
+	require.Equal(t, 4, len(*gens))
```

Comment:
This test is a false positive. It should've only returned `2`.

Reply:
Because the method returns blinding factors for both revealed messages (2) and hidden messages (2), 4 makes sense here